src/Blocks/components/form/assets/form.js

```diff
@@ -649,7 +650,9 @@ export class Form {
 
 	// Dispatch custom event.
 	dispatchFormEvent(element, name) {
-		const event = new CustomEvent(`esForms${name}`);
+		const event = new CustomEvent(`esForms${name}`, {
+			bubbles: true
+		});
```

Comment:
do we have the ensurance this will dispatch after all listeners are set?

Reply:
yes if you put the correct listeners it will